internal/buildsign/ocpbuild/ocpbuildmanager.go

```
@@ -0,0 +1,173 @@
+
+	buildv1 "github.com/openshift/api/build/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/constants"
+)
+
+type Status string
+
+const (
+	ocpbuildTypeBuild = "build"
```

golangci-lint (1.64.8)

const `ocpbuildTypeBuild` is unused

(unused)